qa/statsd/src/cases/04.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env pmpython
+# -*- coding: utf-8 -*-
+
+# Exercises updates of non-config hardcoded metrics:
+# - statsd.pmda.received (before: 0, after: 75)
+# - statsd.pmda.parsed (before: 0, after: 45)
+# - statsd.pmda.aggregated (before: 0, after: 35)
+# - statsd.pmda.dropped (before: 0, after: 40)
+# - statsd.pmda.time_spent_parsing (before: 0, after: non-zero)
+# - statsd.pmda.time_spent_aggregating (before: 0, after: non-zero)
+# - statsd.pmda.metrics_tracked, with its counter, gauge, duration and total instances (before: 0, 0, 0, 0; after: 4, 4, 2, 10)
```

Comment:
I don't know how these tests work, but is this supposed to check also the before/after state?